pdfannots/printer/json.py

```diff
@@ -15,28 +15,19 @@ def annot_to_dict(
 
     result = {
         "type": annot.subtype.name,
-        "page": annot.pos.page.pageno + 1,
+        "page_number" "number": annot.pos.page.pageno + 1,
+        "page_label": getattr(annot.pos.page, 'label', None),
         "start_xy": (annot.pos.x, annot.pos.y),
+        "prior_outline": getattr(doc.nearest_outline(annot.pos), 'title', None),
+        "text": annot.gettext(remove_hyphens) if annot.text else None,
+        "contents": getattr(annot, 'contents', None),
```

Comment:
Simpler example here: why not annot.contents?

Reply:
See above.